nova/virt/vmwareapi/driver.py

```diff
@@ -1027,7 +1031,7 @@ def post_live_migration(self, context, instance, block_device_info,
         """Post operation of live migration at source host."""
         if not migrate_data.is_same_vcenter:
             self._volumeops.delete_shadow_vms(block_device_info, instance)
-        self._vmops.update_cluster_placement(context, instance)
```

Comment:
Did this raise an Exception on trying to update the rules ("configuration is invalid" or something)?

Reply:
It raised `InstanceNotFound` when calling `update_admin_vm_group_membership`, which reasonable considering the VM is gone in a cross-vcenter-vmotion. In the case of staying in the same vcenter, it is a bit redundant, as we do the same on the destination-host.